include/votca/xtp/topology.h

```diff
@@ -36,6 +38,21 @@ class Segment;
  */
 class Topology {
  public:
+
+  typedef Segment::bead_t bead_t;
```

Comment:
what is this?

Reply:
A standardized label for referring to to the type stored in the container, for use with templates.